### PR TITLE
optional timestamp

### PR DESCRIPTION
**Pull request by pivnicek**
So, was thinking something like this to make _timestamps optional.  If you agree, I'll write up the tests and RS version etc...

### CODE REVIEW

**Comment by Halama** [comment]
@pivnicek it look good but some array `$options` parameter whould be better than boolean flag parameter. So then it code it would look like:
```php
$import->import('my-table', $columns, $sourceData, [
  'addTimestamp' => false,
]);
```

**Comment by Halama** [comment]
Also it would be more clear to put this as parameter of `insertAllIntoTargetTable` instead of make it class variable.

**Comment by pivnicek** [comment]
Great!  Agreed.  I'll flush it out.

**Comment by pivnicek** [comment]
I guess I'll add it for `insertOrUpdateTargetTable` also.  If we'll support optional timestamp it should be completely supported imo.

**Comment by Halama** [comment]
@pivnicek ok, add it to `insertOrUpdateTargetTable` please too

**Comment by Halama** [comment]
don't forget to tests, every feature should be developed by tests.

File: src/RedshiftBase.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function __construct(\PDO $connection, $schemaName)
      * @param array CsvFile $csvFiles
      * @return mixed
      */
-    public function import($tableName, $columns, array $sourceData)
+    public function import($tableName, $columns, array $sourceData, array $options = ["useTimestamp" => true])
```
Comment (Halama) [inline src/RedshiftBase.php]: wow, i didn't know this PHP feature:) nice

**Comment by pivnicek** [comment]
OK, this may need a sanity check, but this should cover support for RS and SNFLK full table and incremental loading optional timestamp usage.

**Comment by Halama** [comment]
@pivnicek the tests are failing

**Comment by pivnicek** [comment]
damn, sorry :/  mmt

**Comment by pivnicek** [comment]
that's better

**Pull request merged**